natashasthilaire/Swampy-Sells
# Forgot

### About
This PR enables a user to be directed to the homepage upon inputting the correct username and password, as well as allowing a user to be sent a link to reset their password in the case that they forget it. 

### Test Instructions
1. Run the `forgot` branch
2. Click 'forgot your password'
3. Enter your email address and click the link
4. Input your new password and confirm that you can now login with the correct password

### Screenshots
![image](https://github.com/natashasthilaire/Swampy-Sells/assets/93110503/94204678-9aef-45f6-a022-223ca8d0ae05)

### Closes Tickets
[SS-40](https://swampysells.atlassian.net/browse/SS-40)
